draft-muks-dnsop-dns-catalog-zones.xml

```diff
@@ -699,7 +699,7 @@ please see http://xml.resource.org/authoring/README.html. -->
         zone.</t>
 
         <t>Thanks to Brian Conry, Tony Finch, Evan Hunt, Patrik Lundin, Victoria Risk
-        and Carsten Strettman for reviewing draft proposals and offering comments and
+        and Carsten Strotmann for reviewing draft proposals and offering comments and
```

Comment:
Both spellings are present in this repository, but I'm pretty sure they're the same person...